qa/suites/fs/bugs/multifs_mdsauthcaps/tasks/0-install.yaml

```diff
@@ -0,0 +1,32 @@
+meta:
+- desc: |
+   install ceph/squid v19.2.2
+tasks:
+- install:
+    tag: v19.2.2
```

Comment:
This could be done without doing an upgrade on the ceph cluster. It's just slightly clumsy because `install` task cannot (yet) be applied to selected roles (like `install.upgrade` can). The way to do that is to install v19.2.2 then immediately do `install.upgrade` on the ceph cluster node. After that, you can start the `ceph` task normally.

Reply:
@batrick Thank you for the feedback and ptal. This simplies the commit!

Reply:
New QA Run scheduled with this change using main branch here:

https://pulpito.ceph.com/khiremat-2025-07-08_11:11:38-fs:bugs-main-distro-default-smithi/